ci/Dockerfile.dockerhub

```diff
@@ -0,0 +1,46 @@
+# This Dockerfile is used to create images published to DockerHub. It needs to
+# be kept in sync with the corresponding Dockerfile we use and test in CI
+# (docker/Dockerfile.ubuntu-20).
+
+FROM ubuntu:focal
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+ARG ZEEK_LTS=1
+ARG ZEEK_VERSION=3.0.11-0
```

Comment:
Let's switch this to 4.0.

Reply:
The packages will be ready later today or tomorrow. I'd suggest that we do a more general sweep of setups once that happens, see #827.